manifests/m/Microsoft/PIX/2403.08/Microsoft.PIX.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.4.0 $debug=AUSU.CRLF.5-1-26085-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: "2403.08"
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+Author: Microsoft
+PackageName: Microsoft PIX
+PackageUrl: https://devblogs.microsoft.com/pix/download/
+License: Proprietary
+# LicenseUrl:
+Copyright: Copyright (c) Microsoft Corporation. All rights reserved.
+# CopyrightUrl:
+ShortDescription: Performance tuning and debugging for DirectX 12 games on Windows
+# Description:
+Moniker: pix
+Tags:
+- d3d12
+- dxr
+Agreements:
+- AgreementLabel: End User License Agreement (EULA)
+  AgreementUrl: https://www.microsoft.com/en-us/research/wp-content/uploads/2017/01/Microsoft_Pix_EULA_20160621.pdf
```

Comment:
This EULA is for a different product (a phone app called Pix, see https://en.wikipedia.org/wiki/Microsoft_Pix). Please don't use this!

I don't believe the EULA is available online (it's just in the installer). Is that a requirement for winget?

Reply:
Yes, it must be online in order for it to be required as part of the confirmation flow.

The installer doesn't require the EULA to be accepted when used with the `/passive` switch, so it would be possible to just remove the agreements section here and use the fact that a user implicitly agrees to the EULA by using the software. 

All WinGet installs do state that there is an implicit license they will be agreeing to -
```
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
```

Reply:
I see, thanks. I don't think bypassing the EULA is the right thing to do here.

Let me work on finding a way to publish our EULA somewhere online. I appreciate all your help!

Reply:
Let me know when you do and I'll be happy to update this PR!

Reply:
Okay I've uploaded the EULA at https://aka.ms/pix-eula-april2024!